state/remote/atlas.go

```diff
@@ -48,7 +48,7 @@ func atlasFactory(conf map[string]string) (Client, error) {
 
 	parts := strings.Split(name, "/")
 	if len(parts) != 2 {
-		return nil, fmt.Errorf("malformed name '%s'", name)
+		return nil, fmt.Errorf("malformed name '%s' (valid format is 'username/name')", name)
```

Comment:
Seth, thanks again. 
As a user/operator I would expect the docs to be inclusive of all detail and mention the Atlas remote requires that 2 part string format.
 Now it's up to you if you like the wording I added in the code or decide with something else - either way I'm glad you took the feedback.

Cheers, 
Luis Faustino